tests/plugins/os/unix/bsd/citrix/test__os.py

```diff
@@ -51,15 +53,15 @@ def test_citrix_os(target_citrix: Target, fs_bsd: VirtualFilesystem) -> None:
     assert users[3].home is None
 
     assert users[4].name == "nobody"  # User entry for the nobody user from /etc/passwd
-    assert users[4].home == "/nonexistent"
+    assert users[4].home == posix_path("/nonexistent")
 
     assert users[5].name == "robin"  # Listed in config and /var/nstmp
-    assert users[5].home == "/var/nstmp/robin"
+    assert users[5].home == posix_path("/var/nstmp/robin")
 
     assert users[6].name == "root"  # User entry for /root, from the config
-    assert users[6].home == "/root"
+    assert users[6].home == posix_path("/root")
     assert users[6].shell is None
 
     assert users[7].name == "root"  # User entry for /root, from /etc/passwd
-    assert users[7].home == "/root"
+    assert users[7].home == posix_path("/root")
     assert users[7].shell == "/usr/bin/bash"
```

Comment:
Misc comment that may needs to be it's own PR, but I think `shell` should also be a proper path.

Reply:
Yeah, probably.

It is marked as a string in the record definition. 
Because it is a string, at least it won´t suffer from the issue we had where the path got converted to the flavor of the  host OS. 
But it is semantically clearer to mark it as a path.

Will think about it some more and create a ticket.